deploy/sre-build-test/50-build-test.CronJob.yaml

```
@@ -79,6 +86,12 @@ spec:
                     ;;
                   Complete)
                     echo "$(date): Build Complete"
+                    JOBS_TO_DELETE=$(oc -n "${POD_NS}" get job --no-headers || true | \
+                                                              grep -v "${JOB_NAME}" | \
+                                                              grep -e "${JOB_PREFIX}" -e '0/1' | \
```

This will delete all jobs that match the prefix regardless of status.  Ideally we parse the json to be specific.  But a quick update will match what I think the expectations are, clear the failed jobs so the alerts go away:

```suggestion
                                                              grep "${JOB_PREFIX}" | \
                                                              grep '0/1' | \
```